src/cubic_loader/qlik/ods_qlik.py

```diff
@@ -358,16 +358,17 @@ def cdc_load_folder(self, load_folder: str) -> None:
         try:
             dfm_object = os.listdir(load_folder)[0].replace(".csv.gz", ".dfm").replace("|", "/")
             merge_csv = os.path.join(load_folder, MERGED_FNAME)
+            key_columns = [col["name"].lower() for col in self.etl_status.last_schema if col["primaryKeyPos"] > 0]
+            load_table = f"{self.db_fact_table}_load"
 
             cdc_ts = merge_cdc_csv_gz_files(load_folder)
             self.cdc_verify_schema(dfm_object)
-
-            # Load records into _history table
-            remote_csv_gz_copy(merge_csv, self.db_history_table)
-
             cdc_df = dataframe_from_merged_csv(merge_csv, dfm_object)
 
-            key_columns = [col["name"].lower() for col in self.etl_status.last_schema if col["primaryKeyPos"] > 0]
+            # Load records into _history table
```

Comment:
update comment here. 
q: is `_load` a new type of table at this point? we need to communicate to folks who are using the database what it is, so they can ignore if need be.

Reply:
If it is just a table to load stuff, maybe truncating at the end as well? I want it to be obvious that it shouldn't be used by data analysts.

Reply:
It's sort of a "temp" table. It gets created by the `dmap_import` application when it starts to load a QLIK table and then gets dropped at the end. So DB users would only ever see it while the ETL operations is actively running.